.github/workflows/daily.yml

```
@@ -0,0 +1,28 @@
+name: Node.js daily job
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  NODE_VERSION: 14.x
+
+jobs:
+  build-lto:
+    runs-on: ubuntu-latest
+    # not working on gcc-8 and gcc-9 see https://github.com/nodejs/node/issues/38570
+    container: gcc:11
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Environment Information
+        run: npx envinfo
+      - name: Build lto
+        run: |
+          apt update && apt install ninja-build python-is-python3 -y
```

```suggestion
          apt-get update && apt-get install ninja-build python-is-python3 -y
```

IIRC `apt` is not supposed to be used in automation.